src/__tests__/components/profile/Profile.test.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import { Provider } from "react-redux";
+import store from "../../../store/index";
+import { loadProfile } from "../../../actions/profile/profileActions";
+import Profile from "../../../components/profile/profile";
+
+describe("profile component tests", () => {
+  it("should not regress", () => {
```

Comment:
what does `regress` mean here

Reply:
Good question @AmosWels , It basically confirms that recent program changes or code changes have not adversely affected the functionality of this feature.